sdk/python/feast/infra/gcp.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from typing import List, Optional
+
+from feast import FeatureTable
+from feast.infra.provider import Provider
+from feast.repo_config import DatastoreOnlineStoreConfig
+
+
+def _delete_all_values(client, key) -> None:
+    """
+    Delete all keys under the key path in firestore.
```

Comment:
Nit: `firestore` -> `Firestore`.

Reply:
now that you mention it, it should actually be "datastore" :) let me fix that